pkg/labels/labels.go

```diff
@@ -530,3 +533,15 @@ func LabelSliceSHA256Sum(labels []*Label) (string, error) {
 	}
 	return fmt.Sprintf("%x", sha.Sum(nil)), nil
 }
+
+// generateLabelString generates the string representation of a label with
+// the provided source, key, and value in the format "source:key=value".
+func generateLabelString(source, key, value string) string {
```

Comment:
Isn't there a function that does this?

Reply:
I could create a new Label object and then call String() on that new object, but I felt like that was inefficient.